test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/mq_as_uri_param_for_metadata.test

```diff
@@ -0,0 +1,807 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+mq as URI param for metadata in filters
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create an entity E1 with attr A1 with metadata M1=12
+# 02. Create an entity E2 with attr A1 with metadata M1=13
+# 03. Create an entity E3 with attr A1 with metadata M1=14
+# 04. Lookup all entities having A1.M1==12, see E1
+# 05. Create a subscription for all entities/A1 AND mq=A1.M1==13
+# 06. Dump accumulator, see E2, then reset accumulator
+# 07. Modify E1/A1, setting A1/M1=13
+# 08. Dump accelerator, see E1, then reset accumulator
```

Comment:
accelerator -> accumulator

Check also corresponding test in echoes and expected output. Also in step 10.